packages/terra-toggle-section-header/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-toggle-section-header",
+  "main": "lib/ToggleSectionHeader.js",
+  "version": "0.0.0",
+  "description": "Toggle section header component that transitions content in and out with a click.",
```

Comment:
nit: with a click on the header.

Reply:
Done 0ce0660763557f9776a1936acb4a257e20ec7ce5